### PR TITLE
OSDOCS-9767: 4.13.35 RN

### PR DESCRIPTION
**Pull request by obrown1205**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-9767

Link to docs preview:
https://72114--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes#ocp-4-13-35

QE review:
N/A

### CODE REVIEW

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -3898,4 +3898,24 @@ $ oc adm release info 4.13.34 --pullspecs
...
 [id="ocp-4-13-34-updating"]
 ==== Updating
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-13-35"]
+=== RHSA-2024:0946 - {product-title} 4.13.35 bug fix and security update
+
+Issued: 2024-02-28
+
+{product-title} release 4.13.34, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0946[RHSA-2024:0946] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:0948[RHSA-2024:0948] advisory.
```
Comment (bergerhoffer) [inline release_notes/ocp-4-13-release-notes.adoc:3908]:
```suggestion
{product-title} release 4.13.35, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0946[RHSA-2024:0946] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:0948[RHSA-2024:0948] advisory.
```

**Review by bergerhoffer — COMMENTED** [review]
One typo to fix before we merge!

**Pull request merged**